### PR TITLE
NN's Perceptron

### PR DESCRIPTION
**Pull request by chefdarek**

### CODE REVIEW

**Comment by chefdarek** [comment]
Sprint Challenge

**Review by hughjafro — COMMENTED** [review]
## NN Sprint Challenge

**_Part 1 - Define Terms:_**
- You defined all terms and gave adequate explanations for the concepts.
- Nice use of the word squishify in a complete sentence.

**_Part 2 - Perceptron Implementation:_**
- You implemented a perceptron class, trained it on the training data and updated weights appropriately until the output from the network was very close to the expected training data output (y).

**_Part 3 - MLP Implementation:_**
- Your MLP implementation correctly initializes weights, takes in inputs, combines inputs and weights as a weighted sum, activates the weighted sum, and passes the activated values onto the next layer.
- Backpropagation implementation appropriately updates weights in reverse order and leads to performance improvements in the model.
- Model’s loss should decrease and accuracy improve over multiple iterations.

**_Part 4 - Keras Implementation & Hyperparameter Tuning:_**
- Your Keras implementation appropriately uses a sequential model with dense layers, and appropriate architecture with *at least* _one hidden layer, activation functions, appropriate loss function, and accuracy metric_.
- Model’s verbose output shows significant improvement from first to last epoch during training.
- You showed your work in hyperparameter tuning at least two features and reported the CV accuracy of each parameter combination that is being tested.
